### PR TITLE
Socket interface article fixes

### PR DESCRIPTION
**Pull request by adolfo-rt**
The doc for the status code of the Pickit reply was super outdated. The diff is big because I had to widen the table.

### CODE REVIEW

**Comment by adolfo-rt** [comment]
This PR should be backported to 2.0.

File: docs/robot-integrations/socket/index.rst [hunk 1]
```diff
@@ -247,7 +263,7 @@ Each field is explained below. All **int32** are expressed in Network Byte Form
 |                          |                |   #. KUKA → Euler Angles (z-y’-x”)                                                                                                                     |
 |                          |                |   #. COMAU → Euler Angles (z-y’-z”)                                                                                                                    |
 +--------------------------+----------------+--------------------------------------------------------------------------------------------------------------------------------------------------------+
-| **interface\_version**   | **int32**      | | The version of the robot-Pickit communication.                                                                                                       |
+| **interface\_version**   | **int32**      | | The version of the robot-Pickit communication. The current version is 1.1.                                                                           |
```
Comment (achim-k) [inline docs/robot-integrations/socket/index.rst]:
```suggestion
| **interface\_version**   | **int32**      | | The version of the robot-Pickit communication. The current version is 11.                                                                           |
```

?

Comment (adolfo-rt) [reply docs/robot-integrations/socket/index.rst]: I'll reword it such that it says that the current protocol version is 1.1, which means that we communicate 11.

Comment (achim-k) [reply docs/robot-integrations/socket/index.rst]: Also good :+1:

Comment (adolfo-rt) [reply docs/robot-integrations/socket/index.rst]: Done in 6ab17e3.

**Pull request merged**